src/dds/rs-dds-internal-data.cpp

```diff
@@ -0,0 +1,178 @@
+    }
+    else if( product_id == RS455_PID )
+    {
+        tags.push_back( { RS2_STREAM_COLOR, -1, 1280, 720, RS2_FORMAT_RGB8, 30, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT } );
+        tags.push_back( { RS2_STREAM_DEPTH, -1, 848, 480, RS2_FORMAT_Z16, 30, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT } );
+        tags.push_back( { RS2_STREAM_INFRARED, -1, 848, 480, RS2_FORMAT_Y8, 30, profile_tag::PROFILE_TAG_SUPERSET } );
+        tags.push_back( { RS2_STREAM_GYRO, -1, 0, 0, RS2_FORMAT_MOTION_XYZ32F, 200, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT } );
+        tags.push_back( { RS2_STREAM_ACCEL, -1, 0, 0, RS2_FORMAT_MOTION_XYZ32F, 63, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT } );
+        tags.push_back( { RS2_STREAM_ACCEL, -1, 0, 0, RS2_FORMAT_MOTION_XYZ32F, 100, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT } );
+    }
+    else if( product_id == RS457_PID )
+    {
+        tags.push_back( { RS2_STREAM_COLOR, -1, 640, 480, RS2_FORMAT_RGB8, 30, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT } );
+        tags.push_back( { RS2_STREAM_DEPTH, -1, 640, 480, RS2_FORMAT_Z16, 30, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT } );
+    }
+    else if( product_line == "D400" )
+    {
+        tags.push_back( { RS2_STREAM_DEPTH, -1, 1280, 720, RS2_FORMAT_Z16, 30, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT } );
+        tags.push_back( { RS2_STREAM_INFRARED, 1, 1280, 720, RS2_FORMAT_RGB8, 30, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT } );
+        tags.push_back( { RS2_STREAM_INFRARED, 2, 1280, 720, RS2_FORMAT_RGB8, 30, profile_tag::PROFILE_TAG_SUPERSET } );
+    }
+
+    // For other devices empty tags will be returned, no default profile defined.
+
+    return tags;
+}
+
+
+std::vector< processing_block_factory > dds_rs_internal_data::get_profile_converters( const std::string & product_id,
+                                                                                      const std::string & product_line )
+{
+    std::vector< processing_block_factory > factories;
+    std::vector< processing_block_factory > tmp;
+
+    if( product_line == "D400" )
+    {
+        if( product_id == RS457_PID )
+        {
+            tmp = processing_block_factory::create_pbf_vector< uyvy_converter >( RS2_FORMAT_UYVY,
+                                                                                 target_formats( RS2_FORMAT_UYVY ),
+                                                                                 RS2_STREAM_COLOR );
+            for( auto & it : tmp )
+                factories.push_back( std::move( it ) );
+            tmp = processing_block_factory::create_pbf_vector< uyvy_converter >( RS2_FORMAT_YUYV,
```

Comment:
Shouldn't this be `yuy2_converter`?

Reply:
But isn't it a bug? The source is YUYV, not UYVY... maybe it's a bug in the original, too...

Reply:
Looks like a bug: `uyvy_converter` unpacks UYVY and `yuy2_converter` unpacks YUYV. I.e., it looks like the converter name is the SOURCE format. If we know the source is YUYV, we should use the `yuy2_converter`, right?

Reply:
Remi says it's intended: for D457, YUYV is reported but it's UYVY in actuality.

There may be a bug in the (original) code still: if it's UYVY and `create_pbf_vector` is called with a source of YUYV, then we get a combination where source==dest (YUYV) which generates an identity block rather than UYVY-to-YUYV conversion.

I'm letting this go for this PR as this code will likely be removed soon -- but this might still be an issue for `rs-dds-server`, since it'll send out UYVY as YUYV (meaning the DDS client will get the wrong format).